go2eiko.py

```diff
@@ -4,6 +4,7 @@
 import logging
 from dotenv import load_dotenv
```

Comment:
dotenv will probably be removed to use "true" env variables, which would be easier for the user to setup using Docker.

Reply:
Changed in ed9428c16dbeea8794ec887d62dab078968c52fe (see also https://github.com/eikofee/eikonomiya/pull/28#discussion_r1554844413)